api/API-CONTRACT.md

```diff
@@ -0,0 +1,138 @@
+# BBSTrader API Contract
+
+## Objects
+
+### Account object
+```
+{
+  login: integer
+  balance: float
+  profit: float
+  equity: float
+  server: string
+}
+```
+
+### ClosedOrder object
+```
+{
+  ticket: integer
+  time: milliseconds since epoch
+  order_type: integer
+  volume: float
+  profit: float
+  symbol: string
+  comment: string
+}
+```
+
+### OpenOrder object
+```
+{
+    ticket: integer
+    time_setup: milliseconds since epoch
+    order_type: integer
+    volume_current: float
+    stop_loss: float
+    take_profit: float
+    price_open: float
+    price_current: float
+    symbol: string
+    comment: string
+}
+```
+
+## Endpoints 
+### GET /orders-open
+  Returns all open orders.
+* **URL Params**  
+  None
+* **Data Params**  
+  None
+* **Headers**  
+  Content-Type: application/json  
+* **Success Response:**  
+* **Code:** 200  
+  **Content:**  
+```
+{
+  orders: [
+           {<OpenOrder_Object>},
+           {<OpenOrder_Object>},
+           {<OpenOrder_Object>}
+         ]
+}
+```
+----
+### GET /orders-open/:ticket
+  Returns the open order with the given ticket number.
```

Comment:
Do we have a use case where we want the open order for just one ticket? Right now we are showing a table with all of the open orders and don't have anything like a search functionality where we have users trying to narrow down. (And if we did we could use the same data we have from the GET all response)

Reply:
Works for me